google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerExceptionFactory.java

```diff
@@ -295,8 +307,14 @@ private static SpannerException fromApiException(ApiException exception) {
       code = Status.Code.UNKNOWN;
     }
     ErrorCode errorCode = ErrorCode.fromGrpcStatus(Status.fromCode(code));
-    return SpannerExceptionFactory.newSpannerException(
-        errorCode, exception.getMessage(), exception);
+
+    Throwable exceptionCause = null;
+
+    if (exception.getCause() != null) {
+      exceptionCause = exception.getCause();
+    }
```

Comment:
nit: this block can be removed and replaced with `exception.getCause()` in the method call below